ROADMAP.md

```diff
@@ -1,35 +1,19 @@
 Roadmap
 ---
-This represents the larger, bigger impact features and enhancements we have planned for K3s. There are many more tactical enhancements and fixes that can be found by perusing our [GitHub milestones](https://github.com/rancher/k3s/milestones).
+This represents the larger, bigger impact features and enhancements we have planned for K3s. Features are planned, but do not represent a commitment to develop and can change at any time. There are many more tactical enhancements and fixes that can be found by reviewing our [GitHub milestones](https://github.com/k3s-io/k3s/milestones).
 
-
-v1.19 - End of August 2020
----
-- Add support for config file based configuration
-- Replace experimental embedded dqlite with embedded etcd
-- Support ability to customize baked-in Helm charts
-- Improve and refactor SELinux support
-- Improve CentOS and RHEL 7 and 8 support
-
-
-v1.20 - December 2020
+Next
 ---
-- Support a 2-node hot/cold HA model by leveraging etcd's learner node feature
```

Comment:
Is there no plan about 2-node hot/cold HA model? Thanks.

Reply:
upstream etcd doesn't support this yet. At the moment you can still have only a single learner that is on a short path to full voting member. Multiple learner is documented as being a 3.5 feature, but I have not seen any code for it appear yet:
https://etcd.io/docs/v3.3/learning/learner/#features-in-v35

Reply:
Yes it is, the learner features has no changes except a bug fix: https://etcd.io/blog/2021/announcing-etcd-3.5/

Without the official support, it can only be implemented by the *black magic*, but this may introduce a large complexity.